Jenkinsfile

```diff
@@ -35,9 +35,7 @@ fabric8UITemplate{
                     sh "git pull"
                     sh "git remote set-url origin git@github.com:${project}.git"
 
-                    container('ui'){
```

Comment:
Why did you remove this line?

Reply:
`release.groovy` already adds the container('ui').
this change looks good

Reply:
Oh, okay. I missed that.